### ISSUE TITLE
linter: `import/no-duplicates` false positive

### ISSUE THREAD

**Issue by pumano** [oxc-project/oxc#6736]
### What version of Oxlint are you using?

0.10.1

### What command did you run?

oxlint -c oxlintrc.json --quiet

### What does your `.oxlint.json` config file look like?

```json
{
  "plugins": ["import"],
  "rules": {
    "import/no-duplicates": "error"
  }
}
```

### What happened?

Looks like some regression after #6694 but that not catched, because we don't have some valid test cases for it.

I provide valid test case for reproduction:
```rs
        (
            r"import { ReactiveFormsModule } from '@angular/forms';
            import { RouterModule, Routes } from '@angular/router';",
            None,
        ),
```
